scripts/check-collect-store-take-rates.bash

```
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+SCRIPT_DIR=$(dirname "$0")
+
+collect_store_take_rates() {
+  local output_file="take-rates.yaml"
+
+  set -e
+  echo "Collecting take rates"
+  # Pass-through args (e.g. --from-epoch N for a historical backfill, or --epochs-back N).
+  ${SCRIPT_DIR}/collect-take-rates.bash "$@" > "$output_file"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)